plugins/obs-qsv11/QSV_Encoder_Internal.cpp

```diff
@@ -177,6 +177,23 @@ mfxStatus QSV_Encoder_Internal::Open(qsv_param_t *pParams, enum qsv_codec codec)
 	return sts;
 }
 
+PRAGMA_WARN_PUSH
+PRAGMA_WARN_DEPRECATION
+static inline bool IsOptimizedBRCSupport(const mfxPlatform &platform,
+					 mfxU16 rateControl)
+{
+#if defined(_WIN32)
```

Comment:
This seems a bit like a sledgehammer and will prevent any non-Windows platform from being able to utilize this even if they are building against libvpl 2.12 or newer. Would it not be more robust to use the `MFX_VERSION` defines?
```suggestion
#if MFX_VERSION_MAJOR >= 2 && MFX_VERSION_MINOR >= 12
```
or:
```suggestion
#if MFX_VERSION >= 2012
```

Unless there is a driver/runtime requirement as well.